content/en/project/general-spinal-cord-segmentation/index.md

```diff
@@ -0,0 +1,136 @@
+
+My goal is to build a more general model to automatically perform spinal cord segmentation across contrasts, vendors and pathologies.
+
+Below is a visual example of the segmentation result on one subject using a contrast outside of the training scheme comparing the ground truth or manual segmentation (left) with the aggregated model (center) and one of the specific models (right).
+
+![Visual result example](result_basel_visual.gif)
+
+### Tools
+
+Main tools: Python/Pytorch/Git/ssh/Jupyter Notebooks
+
+Most important: [nnU-Net](https://github.com/MIC-DKFZ/nnUNet) -> a state-of-the-art “self-configuring method for deep learning-based biomedical image segmentation”.
+
+
+### Methodology
+Train and compare 2 frameworks (individual models vs. general model) for performing automatic spinal cord segmentation.
+
+![Frameworks overview](frameworks.png)
+
+### Data
+
+Various (6) Spinal Cord MRI datasets available at Neuropoly with manual segmentation.
+
+Our training datasets:
+* Spine Generic Public Database* (multi-subject), n=244, 3 vendors, 6 contrasts (T2w, T1w, T2*w, 3D axial GRE (MTon), GRE-T1w (MToff), DWI), pathology: Mild Compression, Healthy Controls.
+* INSPIRED, n=79, 1 contrast (acq-cspineAxial_T2w), pathology: Degenerative Cervical Myelopathy (38), Spinal Cord Injury, Healthy Controls.
+* SCI-Colorado, n=80, 1 contrast (T2w), pathology: Spinal Cord Injury.
+* Canproco, n=445, 1 contrast (T2w), pathology: MS.
+
+Test datasets (outside training distribution):
+* Basel-mp2rage, n=283, 1 contrast (MP2RAGE "UNI"), pathology: Multiple Sclerosis (180), Healthy Controls (103).
+* Extra: [Rohan](https://github.com/brainhack-school2023/banerjee_project)’s fMRI test set, n=8, 1 contrast (EPI-GRE).
+
+
+**Cohen-Adad, J., Alonso-Ortiz, E., Abramovic, M., Arneitz, C., Atcheson, N., Barlow, L., Barry, R.L., Barth, M., Battiston, M., Büchel, C., et al.: Open-access quantitative mri data of the spinal cord and reproducibility across participants, sites and manufacturers. Scientific data 8(1), 219 (2021)
+
+### Objectives
+
+* Create a script for converting N BIDS datasets into one large “nnU-Net-ready” dataset train/test split. Read more [here](https://github.com/brainhack-school2023/bouchard_project/tree/main/docs).
+* Compare a general model to specific models for spinal cord segmentation on the datasets’ test sets. Results shown [here](https://github.com/brainhack-school2023/bouchard_project/blob/main/results/results_analysis.ipynb).
+* Test the resulting 2 frameworks on a fifth and sixth dataset to demonstrate the generalizability capabilities of the larger model. Results shown [here](https://github.com/brainhack-school2023/bouchard_project/blob/main/results/results_analysis.ipynb).
+
+### Deliverables
+
+* A Github repository with codes and scripts to reproduce training and testing `github.com/brainhack-school2023/bouchard_project`.
```

Comment:
```suggestion
* A [Github repository](https://github.com/brainhack-school2023/bouchard_project) with codes and scripts to reproduce training and testing.
```

Reply:
There are several references to docs throughout the report which would be better formatted as links. If you want to have the links in plain text as well, you can make links like [github.com/bla](https://github.com/bla]. These will be rendered in the same style as what you used, but they will be clickable.